src/consensus.rs

```diff
@@ -337,7 +332,13 @@ impl Consensus {
             }
             Ok(Message::FromClient(message)) => {
                 log::debug!("Proposing entry from client with length: {}", message.len());
-                self.node.propose(vec![], message)?
+                match self.node.propose(vec![], message) {
+                    Ok(_) => {}
+                    Err(consensus_err) => {
+                        // Do not stop consensus if client proposal failed.
+                        log::error!("Failed to propose entry: {:?}", consensus_err);
+                    }
+                }
```

Comment:
This part is debatable, not sure if we need to propagate this error if it was just a delaying node. @e-ivkov

Reply:
Yes, I agree that we can just log it if client proposal failed